x-pack/plugins/maps/public/kibana_services.d.ts

```diff
@@ -30,6 +30,9 @@ export function getUiActions(): any;
 export function getCore(): any;
 export function getNavigation(): any;
 export function getCoreI18n(): any;
+export function getSearchService(): any;
```

Comment:
Can you remove this getter. Not sure we want to expose this since getSearchSource gets what is needed in Maps and getSearchService is to similar in name and confusing as to which should be used.

Reply:
np, removed